enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/RegularCatchupServerHandler.java

```diff
@@ -99,22 +97,21 @@ public ChannelHandler getStoreIdRequestHandler( CatchupServerProtocol catchupSer
     @Override
     public ChannelHandler storeListingRequestHandler( CatchupServerProtocol catchupServerProtocol )
     {
-        return new PrepareStoreCopyRequestHandler( catchupServerProtocol, checkPointerSupplier, storeCopyCheckPointMutex, dataSourceSupplier,
-                new PrepareStoreCopyFilesProvider( pageCache, fs ) );
+        return new PrepareStoreCopyRequestHandler( catchupServerProtocol, checkPointerServiceBundle::getCheckPointer, storeCopyCheckPointMutex,
```

Comment:
This changes from sending in a supplier to sending in the checkpointer in this instance. I think you should send in the checkPointerServiceBundle

Reply:
This comment is wrong. Sorry